Source/WebCore/html/HTMLMarqueeElement.cpp

```diff
@@ -119,7 +119,7 @@ void HTMLMarqueeElement::collectPresentationalHintsForAttribute(const QualifiedN
         break;
     case AttributeNames::scrolldelayAttr:
         if (!value.isEmpty())
-            addPropertyToPresentationalHintStyle(style, CSSPropertyWebkitMarqueeSpeed, limitToOnlyHTMLNonNegative(value, RenderStyle::initialMarqueeSpeed()), CSSUnitType::CSS_MS);
+            addPropertyToPresentationalHintStyle(style, CSSPropertyWebkitMarqueeSpeed, limitToOnlyHTMLNonNegative(value, 85), CSSUnitType::CSS_MS);
```

Comment:
Surprised to see a literal number here rather than a named constant. Often we host such values to the top of the file?

Reply:
Yeah, good point. Eventually they can probably just be in the IDL as that is where they are defined:

https://html.spec.whatwg.org/multipage/obsolete.html#the-marquee-element